### PR TITLE
184521759 team selection

### PR DESCRIPTION
**Pull request by npraderi**

### CODE REVIEW

File: src/state/recoil/selectors/teamWorkingHoursSelector.ts
[the remark this answers is not in the document]
Comment (npraderi) [reply src/state/recoil/selectors/teamWorkingHoursSelector.ts:1]: You are right

File: src/hooks/useHover.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      if (element) {
+        element.addEventListener("mouseover", handleMouseOver);
+        element.addEventListener("mouseout", handleMouseOut);
+        return () => {
+          element.removeEventListener("mouseover", handleMouseOver);
+          element.removeEventListener("mouseout", handleMouseOut);
+        };
+      }
+    },
+    [value] // Recall only if ref changes
```
Comment (juanlarrambebere) [inline src/hooks/useHover.ts]: Do you need to re-run this effect when the value changes? Do you use the value inside the effect? 

Also, the comment should be removed 😉

Comment (npraderi) [reply src/hooks/useHover.ts]: I mix the cases

**Review by juanlarrambebere — DISMISSED** [review]
Looks good. Make sure to solve conflicts and :shipit:

**Pull request merged**